.github/workflows/release.yml

```diff
@@ -1,6 +1,12 @@
 name: Release
 
 on:
+  workflow_call:
+  secrets:
```

Comment:
I wanted to find out a way to default these, but only inputs can be defaulted. [docs](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#onworkflow_callinputs)

Reply:
Oh nice! So then we'd use [this](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idsecrets) when calling the reusable action?

Reply:
Yes, exactly!